i18n/ja.json

```diff
@@ -301,27 +301,27 @@
   },
   {
     "id": "api.channel.post_update_channel_displayname_message_and_forget.create_post.error",
-    "translation": "表示名更新メッセージ %v を投稿できませんでした"
+    "translation": "表示名更新メッセージを投稿できませんでした"
   },
   {
     "id": "api.channel.post_update_channel_displayname_message_and_forget.retrieve_user.error",
-    "translation": "チャンネル表示名更新メッセージ%vを保存する際にユーザーを取得できませんでした"
+    "translation": "チャンネル表示名を更新する際にユーザーを取得できませんでした"
   },
   {
     "id": "api.channel.post_update_channel_displayname_message_and_forget.updated_from",
     "translation": "%sがチャンネル表示名を %s から %s に更新しました"
   },
   {
     "id": "api.channel.post_update_channel_header_message_and_forget.post.error",
-    "translation": "Failed to post update channel header message"
+    "translation": "チャンネルヘッダー更新メッセージを投稿できませんでした"
   },
   {
     "id": "api.channel.post_update_channel_header_message_and_forget.removed",
     "translation": "%sはチャンネルヘッダーを削除しました(これまでは%sでした)"
   },
   {
     "id": "api.channel.post_update_channel_header_message_and_forget.retrieve_user.error",
-    "translation": "チャンネルのヘッダー更新メッセージ%vを保存する際にユーザーを取得できませんでした"
+    "translation": "チャンネルのヘッダーメッセージ%vを更新する際にユーザーを取得できませんでした"
```

Comment:
He is totally right, the are no parameters given for that message, we should submit a PR to fix that, and then it can be fixed on the translation server for the next translations PR